src/com/palmergames/bukkit/towny/object/economy/VaultEconomyAdapter.java

```diff
@@ -0,0 +1,70 @@
+package com.palmergames.bukkit.towny.object.economy;
+
+import net.milkbowl.vault.economy.Economy;
+import net.milkbowl.vault.economy.EconomyResponse;
+import org.bukkit.World;
+
+@SuppressWarnings("deprecation")
+public class VaultEconomyAdapter implements EconomyAdapter {
+	
+	final Economy economy;
+	
+	public VaultEconomyAdapter(Economy economy) {
+		this.economy = economy;
+	}
+
+	@Override
+	public boolean add(String accountName, double amount, World world) {
+		return economy.depositPlayer(accountName, amount).type == EconomyResponse.ResponseType.SUCCESS;
+	}
+
+	@Override
+	public boolean subtract(String accountName, double amount, World world) {
+		return economy.withdrawPlayer(accountName, amount).type == EconomyResponse.ResponseType.SUCCESS;
+	}
+
+	@Override
+	public boolean hasAccount(String accountName) {
+		return economy.hasAccount(accountName);
+	}
+
+	@Override
+	public double getBalance(String accountName, World world) {
+		return economy.getBalance(accountName);
+	}
+
+	@Override
+	public void newAccount(String accountName) {
+		economy.createPlayerAccount(accountName);
+	}
+
+	@Override
+	public void deleteAccount(String accountName) {
+		// Attempt to zero the account as Vault provides no delete method.
+		if (!economy.hasAccount(accountName))
+			economy.createPlayerAccount(accountName);
```

Comment:
Also, I am not sure why an account needs to be created if it doesn't exist, if the end-goal is to mimic deleting the account...